run.py

```diff
@@ -3,16 +3,15 @@
 
 from structlog import wrap_logger
 
-
-if not os.getenv('APP_SETTINGS'):
-    os.environ['APP_SETTINGS'] = 'DevelopmentConfig'
-
-from response_operations_ui import app  # NOQA # pylint: disable=wrong-import-position
+from response_operations_ui import create_app
 
 
 logger = wrap_logger(logging.getLogger(__name__))
 
 
 if __name__ == '__main__':
+    if not os.getenv('APP_SETTINGS'):
```

Comment:
This should be ok to default to dev right because CF will call create_app() in CF?

Reply:
Precisely 👍 CF won't be using this file. Only docker and local dev.